docs: Search Metadata Batch 05 (#14621)
hashicorp/consul: website/content/docs/k8s/operations/certificate-rotation.mdx

---
layout: docs
page_title: Rotate TLS Certificates for Consul on Kubernetes
description: >-

> As of Consul Helm version 0.29.0, new server agent TLS certificates are issued every time the Helm version is upgraded. Learn how to trigger certificate rotation manually when it doesn't happen automatically.

Reviewer: ```suggestion
In Consul Helm version 0.29.0 and later, new server agent TLS certificates are issued every time the Helm version is upgraded. Learn how to manually trigger certificate rotation if they do not rotate automatically.
```
I wonder if we should cut the intro statement about the version.
